ui/src/pages/ShoeMap.tsx

```diff
@@ -3,6 +3,7 @@ import styled from "styled-components";
 import { Map, TileLayer } from "react-leaflet";
 import "leaflet/dist/leaflet.css";
 
+import { Pin, PinState } from "../components/Pin";
```

Comment:
Delete unused import

Reply:
we have to use it to render pin that have the unfocused and selected state, the default state is resting.